### PR TITLE
Update index.md for Error Pages like 500 Internal Server Error

### PR DESCRIPTION
**Pull request by idseefeld**
I figured out a simple solution for e.g. 500 internal server error. As the mentioned Microsoft documentation is not very helpful regarding Umbraco specifics, I think this example will be valuable for some people.  Yours,
Dirk

### CODE REVIEW

File: Tutorials/Custom-Error-Pages/index.md [hunk 1]
```diff
@@ -128,3 +128,67 @@ If you set up everything correctly and the error pages are not showing correctly
 ## Handling errors in ASP.NET Core
 
 For common approaches to handling errors in ASP.NET Core web apps, see the [Handle errors in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?view=aspnetcore-6.0) article in the Microsoft Documentation.
+
+### Basic Example
```
Comment (sofietoft) [inline Tutorials/Custom-Error-Pages/index.md:132]:
```suggestion
## 500 errors
```

File: Tutorials/Custom-Error-Pages/index.md [hunk 2]
```diff
@@ -128,3 +128,67 @@ If you set up everything correctly and the error pages are not showing correctly
...
+
+A basic solution in four steps:
```
Comment (sofietoft) [inline Tutorials/Custom-Error-Pages/index.md:134]:
```suggestion
The following steps guides you through setting up a page for internal server errors (500 errors).
```

File: Tutorials/Custom-Error-Pages/index.md [hunk 3]
```diff
@@ -128,3 +128,67 @@ If you set up everything correctly and the error pages are not showing correctly
...
+
+#### First Step
+
+1. In your Umbraco web project create a folder `~/controllers` containing a file called `ErrorController.cs` with this content
```
Comment (sofietoft) [inline Tutorials/Custom-Error-Pages/index.md:138]:
```suggestion
1. Create a `~/controllers` folder in your Umbraco web project.
2. Create a file in this folder, called `ErrorController.cs`.
3. Add the following code to the file:
```

File: Tutorials/Custom-Error-Pages/index.md [hunk 4]
```diff
@@ -128,3 +128,67 @@ If you set up everything correctly and the error pages are not showing correctly
...
+    ```json
+    "Umbraco": {
+    "CMS": {
+        "Global": {
+        "ReservedPaths": "~/app_plugins/,~/install/,~/mini-profiler-resources/,~/umbraco/,~/error/",
+        ...
+    ```
+
+3. Create the redirect pages from 1. step as regular content nodes in the backoffice. They should neither appear in navigation menus or sitemaps. In this example you would create under root node `Statuscodes` with a subnode `500`.
+4. Update the `Configure`method in file `Startup.cs`
```
Comment (sofietoft) [inline Tutorials/Custom-Error-Pages/index.md:176]:
```suggestion
4. Update the `Configure` method in file `Startup.cs` 
```

**Review by sofietoft — COMMENTED** [review]
Thanks for adding this guide to our docs @idseefeld ! Much appreciated!

I've added some suggestions.
I think it's important that we try to keep the steps in our tutorials short and concise. Take a look at let me know what you think! 🤞 

I'll make sure we get this tested as well, as soon as possible!

**Comment by idseefeld** [comment]
I have copied the changes into https://github.com/umbraco/UmbracoDocs/pull/4775

**Pull request closed without merging**